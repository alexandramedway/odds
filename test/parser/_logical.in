do !true
do !false
do true && false
do true || false
do !(true || false)
