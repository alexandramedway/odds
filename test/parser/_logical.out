[
    Do(Unop(Not, Bool_lit(true))) ;
    Do(Unop(Not, Bool_lit(false))) ;
    Do(Binop(Bool_lit(true), And, Bool_lit(false))) ;
    Do(Binop(Bool_lit(true), Or, Bool_lit(false))) ;
    Do(Unop(Not, Binop(Bool_lit(true), Or, Bool_lit(false))))
]
